api/openapi.yaml

```diff
@@ -1945,6 +2300,39 @@ components:
             type: string
           nullable: true
 
+    AssetScanEstimationState:
+      type: object
+      properties:
+        state:
+          type: string
+          enum:
+            - Pending
+            - Aborted
+            - Failed
+            - Done
+        stateMessage:
+          description: Human-readable message indicating details about the last state transition.
+          type: string
+        stateReason:
+          description: Machine-readable, UpperCamelCase text indicating the reason for the condition's last transition.
+          type: string
+          enum:
+            - Aborted
+            - TimedOut
+            - OneOrMoreAssetFailedToEstimate
+            - DiscoveryFailed
```

Comment:
`DiscoveryFailed` is related here?

Reply:
not relevant and not used. can be removed